woocommerce/pinterest-for-woocommerce
# Fix setup tracking link

### Changes proposed in this Pull Request:

Closes #262 

* Added a [new link reference](https://help.pinterest.com/en/business/article/install-the-pinterest-tag) in `includes/admin/class-pinterest-for-woocommerce-admin.php` based on the issue indications. 
* Change the name from `setupTracking` to `installTags` following issue indications.  

### Detailed test instructions:

1. Go to Onboarding - Setup tracking  `admin.php?page=wc-admin&path=%2Fpinterest%2Fonboarding&step=setup-tracking`
2. Click on the Read More link
3. You should see Install Pinterest tag docs in a new tab.
